### PR TITLE
Add missing paramenters to Material serializer and MaterialLoader

### PR DESCRIPTION
**Pull request by tentone**
Hi

This PR adds some missing parameters to the Material.toJSON and MaterialLoader.
 - skinning
 - morphNormals
 - envReflectivity
 - ...

Cumps

### CODE REVIEW

File: src/loaders/MaterialLoader.js [hunk 1]
```diff
@@ -73,15 +73,20 @@ Object.assign( MaterialLoader.prototype, {
...
 		if ( json.colorWrite !== undefined ) material.colorWrite = json.colorWrite;
+		
 		if ( json.wireframe !== undefined ) material.wireframe = json.wireframe;
 		if ( json.wireframeLinewidth !== undefined ) material.wireframeLinewidth = json.wireframeLinewidth;
 		if ( json.wireframeLinecap !== undefined ) material.wireframeLinecap = json.wireframeLinecap;
 		if ( json.wireframeLinejoin !== undefined ) material.wireframeLinejoin = json.wireframeLinejoin;
+
 		if ( json.morphTargets !== undefined ) material.morphTargets = json.morphTargets;
+		if ( json.morphNormals !== undefined ) material.morphNormals = json.morphNormals;
+		if ( json.skinning !== undefined ) material.skinning = json.skinning;
```
Comment (mrdoob) [inline src/loaders/MaterialLoader.js]: `material.skinning` was recently removed. 118fedfb87714f3702627b099badc8cb7911431c

**Comment by tentone** [comment]
@mrdoob Yap, sorry my bad missed that, removed skinning from the serialization.

**Comment by tentone** [comment]
Solved conflict, with skinning attribute.

**Comment by Mugen87** [comment]
#13904, #14833 and #14877 and  added most properties of this PR. `refractionRatio` and `morphNormals` are still missing though. Rest is done here #16876

**Pull request closed without merging**